src/reporting/interfaces/mappingInterfaces/CustumCalculations.ts

```diff
@@ -0,0 +1,137 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Typo in file name. Should be Cust**o**mCalculations.ts

Reply:
fixed typo